[Snyk] Fix for 1 vulnerabilities

mod opened the pull request:
![snyk-top-banner](https://redirect.github.com/andygongea/OWASP-Benchmark/assets/818805/c518c423-16fe-447e-b67f-ad5a49b5d123)

### Snyk has created this PR to fix 1 vulnerabilities in the rubygems dependencies of this project.

#### Snyk changed the following file(s):

- `Gemfile`



<details>
<summary>⚠️ <b>Warning</b></summary>

```
Failed to update the Gemfile.lock, please update manually before merging.
```

</details>



#### Vulnerabilities that will be fixed with an upgrade:

|  | Issue | Score | 
:-------------------------:|:-------------------------|:-------------------------
![high severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/h.png 'high severity') | Relative Path Traversal <br/>[SNYK-RUBY-RACK-9398129](https://snyk.io/vuln/SNYK-RUBY-RACK-9398129) | &nbsp;&nbsp;**721**&nbsp;&nbsp; 




---

> [!IMPORTANT]
>
> - Check the changes in this PR to ensure they won't cause issues with your project.
> - Max score is 1000. Note that the real score may have changed since the PR was raised.
> - This PR was automatically created by Snyk using the credentials of a real user.

---

**Note:** _You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs._

For more information: <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiIyMThlMmE0ZS01ODYxLTRkNmYtOTVlOC1kNDNlNGU3ODM3M2IiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6IjIxOGUyYTRlLTU4NjEtNGQ2Zi05NWU4LWQ0M2U0ZTc4MzczYiJ9fQ==" width="0" height="0"/>
🧐 [View latest project report](https://app.snyk.io/org/mod/project/478f2032-55a5-4895-864a-35b37f77eaa9?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)
📜 [Customise PR templates](https://docs.snyk.io/scan-using-snyk/pull-requests/snyk-fix-pull-or-merge-requests/customize-pr-templates?utm_source=github&utm_content=fix-pr-template)
🛠 [Adjust project settings](https://app.snyk.io/org/mod/project/478f2032-55a5-4895-864a-35b37f77eaa9?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)
📚 [Read about Snyk's upgrade logic](https://docs.snyk.io/scan-with-snyk/snyk-open-source/manage-vulnerabilities/upgrade-package-versions-to-fix-vulnerabilities?utm_source=github&utm_content=fix-pr-template)

---

**Learn how to fix vulnerabilities with free interactive lessons:**

🦉 [Relative Path Traversal](https://learn.snyk.io/lesson/directory-traversal/?loc&#x3D;fix-pr)

[//]: # 'snyk:metadata:{"customTemplate":{"variablesUsed":[],"fieldsUsed":[]},"dependencies":[{"name":"grape","from":"1.3.3","to":"2.1.0"},{"name":"rails","from":"5.2.4.4","to":"7.0.0"},{"name":"vault-rails","from":"0.6.0","to":"0.7.1"}],"env":"prod","issuesToFix":["SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129","SNYK-RUBY-RACK-9398129"],"prId":"218e2a4e-5861-4d6f-95e8-d43e4e78373b","prPublicId":"218e2a4e-5861-4d6f-95e8-d43e4e78373b","packageManager":"rubygems","priorityScoreList":[721],"projectPublicId":"478f2032-55a5-4895-864a-35b37f77eaa9","projectUrl":"https://app.snyk.io/org/mod/project/478f2032-55a5-4895-864a-35b37f77eaa9?utm_source=github&utm_medium=referral&page=fix-pr","prType":"fix","templateFieldSources":{"branchName":"default","commitMessage":"default","description":"default","title":"default"},"templateVariants":["pr-warning-shown","priorityScore"],"type":"auto","upgrade":["SNYK-RUBY-RACK-9398129"],"vulns":["SNYK-RUBY-RACK-9398129"],"patch":[],"isBreakingChange":true,"remediationStrategy":"vuln"}'

yivo commented:
@mod 

Finished in 4 minutes 26.1 seconds (files took 8.34 seconds to load)
573 examples, 0 failures, 10 pending

yivo commented:
@mod 

Finished in 3 minutes 41.9 seconds (files took 9.08 seconds to load)
573 examples, 0 failures, 10 pending

yivo commented:
@mod ping

snyk-bot committed:
fix: Gemfile to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-RUBY-RACK-9398129
